OutOfSchool/OutOfSchool.WebApi/Services/SocialGroupService.cs

```diff
@@ -96,11 +95,18 @@ public async Task<SocialGroupDto> Update(SocialGroupDto dto, LocalizationType lo
     {
         logger.LogInformation($"Updating SocialGroup with Id = {dto?.Id}, {localization} localization, started.");
 
+        if (dto == null)
+        {
+            logger.LogError("Updating failed. The provided SocialGroupDto is null.");
+
```

Comment:
it should be moved to the top of the method, and maybe, changed to throw exception unless there is a reason why we use null as the error handling result.